src/jit/decomposelongs.cpp

```diff
@@ -920,15 +925,24 @@ GenTree* DecomposeLongs::DecomposeShift(LIR::Use& use)
 
     GenTree* tree   = use.Def();
     GenTree* gtLong = tree->gtGetOp1();
+    GenTree* oldShiftOp = tree->gtGetOp2();
+
```

Comment:
Do we actually need this, since we have shiftWidthOp, below, which looks like the same thing?